### DISCUSSION TITLE
FATROP interface missing?

### DISCUSSION THREAD

**Discussion by gelatinouscube42** [casadi/casadi#3771] [Q&A] [answered]
All,

I'm trying to test the performance of FATROP against an application I have, but the interface plugins seem to be missing from the casadi version I've pulled from pip (3.6.5).

Do I have to compile these on my own, or is there a particular distribution (prerelease maybe?) that I have to pull?

Thanks!

**Comment by tmmsartor** [comment, marked as answer]
Yes, it has been said that fatrop interface will be shipping from version 3.6.6, but you can already try it using nightly build here: https://github.com/casadi/casadi/releases/tag/nightly-fatrop_nlp.
One way is to download the right .whl (it is a python wheel) file for your system and install it with `pip install file.whl`

**Reply by gelatinouscube42** [reply]
Neat, thanks for the tip!
